lib/genres.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+
+class Genres extends StatelessWidget {
+  final Movie _mov;
```

Comment:
the underscore is reserved for private implementations in dart, don't use it on constructors parameters and use the full verbose on the attributes to make the code more representative

Reply:
done